packages/server/tsconfig.build.json

```diff
@@ -2,7 +2,11 @@
   "extends": "../../tsconfig.build.json",
   "compilerOptions": {
     "declarationDir": "dist",
-    "types": ["node"]
+    "types": ["node"],
+    // https://github.com/tsconfig/bases/blob/main/bases/node18.json
+    "lib": ["es2023", "DOM", "DOM.Iterable"],
+    "module": "ES2020", // <-- this is different from the base
```

Comment:
I think this is fine 🤷

Reply:
module should almost always be "Preserve" when using a bundler. Why did you opt for this here?

Reply:
because we use it for rollup now and they need a specific `module`